opus/application/static_media/js/browse.js

```diff
@@ -1416,22 +1432,30 @@ var o_browse = {
         // don't delete the metadata if the observation is in the cart
         if (!galleryObsElem.eq(index).hasClass("in")) {
             let delOpusId = galleryObsElem.eq(index).data("id");
+            if ($("#galleryView").hasClass("show")) {
```

Comment:
Should rename these classes at some point to op-gallery-view and op-gallery-view-contents

Reply:
we discussed and decided to leave the rename for a different pull request.